recipes/nodejs/config.yml

```diff
@@ -5,3 +5,5 @@ versions:
     folder: all
   "16.3.0":
     folder: all
+  "18.15.0":
```

Comment:
Would it make sense to package `18.16.0` to reduce the amount of versions in cci?